api/serverpb/json/client/server/logs_parameters.go

```diff
@@ -111,13 +118,40 @@ func (o *LogsParams) SetHTTPClient(client *http.Client) {
 	o.HTTPClient = client
 }
 
+// WithPprof adds the pprof to the logs params
+func (o *LogsParams) WithPprof(pprof *bool) *LogsParams {
+	o.SetPprof(pprof)
+	return o
+}
+
+// SetPprof adds the pprof to the logs params
+func (o *LogsParams) SetPprof(pprof *bool) {
+	o.Pprof = pprof
+}
+
 // WriteToRequest writes these params to a swagger request
 func (o *LogsParams) WriteToRequest(r runtime.ClientRequest, reg strfmt.Registry) error {
 	if err := r.SetTimeout(o.timeout); err != nil {
 		return err
 	}
 	var res []error
 
+	if o.Pprof != nil {
+
+		// query param pprof
+		var qrPprof bool
+
+		if o.Pprof != nil {
+			qrPprof = *o.Pprof
+		}
+		qPprof := swag.FormatBool(qrPprof)
+		if qPprof != "" {
```

Comment:
`swag.FormatBool` can return only `true` or `false`. No need to check for emptiness